src/systems/menu_defn.cpp

```diff
@@ -1007,17 +988,7 @@ static Widget s_hide_widgets[] = {
 };
 
 static const char* s_timer_types[] = {"Default", "Freeze at max", "Freeze at 0", "Count up from 0"};
-
-static Widget s_bouncy_fallout_widget[] = {
-    {
-        .type = WidgetType::Checkbox,
-        .checkbox =
-            {
-                .label = "Bouncy Fallout Plane",
-                .pref = pref::BoolPref::BouncyFalloutPlane,
-            },
-    },
-};
+static const char* s_fallout_plane_type[] = {"Normal", "Disabled", "Bouncy"};
```

Comment:
`FALLOUT_PLANE_TYPES`

Reply:
All of the other const char arrays in menu_defn.cpp use the formatting style I used, do you think they should all be changed?

Reply:
Honestly yes, they're constants, not mutable arrays

Reply:
Alrighty, I'll update them